fapi-client/src/test/scala/com/gu/facia/api/TestModel.scala

```diff
@@ -185,6 +185,7 @@ object TestModel {
     tags: Seq[StubTag],
     elements: Option[Seq[StubElement]]
   ) extends Content {
+    val channels: Option[Seq[ContentChannel]] = None // alternatively, create StubContentChannel
```

Comment:
This is to cope with the channels model changes introduced in https://github.com/guardian/content-api-models/pull/211, which was released with `content-api-models` [v17.6.0](https://github.com/guardian/content-api-models/releases/tag/v17.6.0) in April 2023.

As none of the tests in `facia-scala-client` seem to care about the value of `channels`, its easiest to just make this value `None` rather than create a `StubContentChannel` - but if in the future tests do eventually need this value, we can make one.